tsconfig.json

```
@@ -21,7 +21,7 @@
     "noUnusedParameters": true,
     "pretty": true,
     "noErrorTruncation": true,
-    "noImplicitAny": true,
+    "noImplicitAny": false,
```

```suggestion
    "noImplicitAny": true,
```
This setting would lower the type safety of the entire repo by allowing types that aren't inferred or explicitly typed to be `any` (which disables typechecking). We should re-enable this, but let me know if you need help with any TypeScript errors as a result.